src/Groups/abelian_aut.jl

```diff
@@ -195,14 +203,18 @@ function defines_automorphism(G::TorQuadModule, M::ZZMatrix)
   return true
 end
 
-function Base.show(io::IO, ::MIME"text/plain", f::AutomorphismGroupElem{T}) where T<:TorQuadModule
+function Base.show(io::IO, ::MIME"text/plain", f::AutomorphismGroupElem{TorQuadModule})
   D = domain(parent(f))
   io = pretty(io)
-  println(IOContext(io, :compact => true), "Isometry of ", Lowercase(), D, " defined by")
-  print(io, Indent(), matrix(f), Dedent())
+  println(io, "Isometry of")
+  println(IOContext(io, :compact => true), Indent(), Lowercase(), D, Dedent())
+  println(io, "with matrix representation")
+  print(io, Indent())
+  show(io, MIME"text/plain"(), matrix(f))
```

Comment:
Why is the detailed printing needed here? Strictly speaking it is against our rules (but I'd be happy to be flexible about that, but for that it'd be good to know why this is beneficial here, over 
```suggestion
  show(io, matrix(f))
```
In the examples I looked at I did not see a difference. If you are aware of examples where it matters, perhaps we could add at least one of those as a test?



Otherwise, this PR looks fine to me (and again: I am not against this line, but I'd like to understand it :-) )

Reply:
OK, that makes sense to me (and as I said, I am not opposed, I just want to understand, which I now do, thanks).